src/olympia/pages/templates/pages/dev_faq.html

```diff
@@ -394,8 +394,8 @@ <h2 id="license">{{ _('References for Open Source Licenses') }}</h2>
     </p>
 
     <h3>
-      <a href="https://techbase.kde.org/Policies/Licensing_Policy">
-        https://techbase.kde.org/Policies/Licensing_Policy
+      <a href="https://community.kde.org/Policies/Licensing_Policy">
+        https://community.kde.org/Policies/Licensing_Policy
```

Comment:
Ah, I commented on the issue before I saw this PR. See https://github.com/mozilla/addons-server/issues/5465#issuecomment-303209402